### ISSUE TITLE
cb is not a function when saving file

### ISSUE THREAD

**Issue by davej** [thesbros/gulp-run-electron#1]
Seems to happen when I save a file and the `rerun` function is called.

Error I'm getting is:

```
/Users/dave/my-app/node_modules/gulp-run-electron/src/index.js:18
            cb(null, file);
   ^
TypeError: cb is not a function
    at done (/Users/dave/my-app/node_modules/gulp-run-electron/src/index.js:18:4)
```

After inspecting `cb`, it's value at the point of failure is:

```
{ type: 'changed',
  path: '/Users/dave/my-app/services/js/uninstall.js' }
```

But if I remove the `uninstall.js` file then it will still fail (`cb` will be an object with a different path).

**Comment by davej** [comment]
Adding the following to index.js:17 (check if `cb` is a function) seems to fix it but I'm not sure if it's correct:

```
if (typeof cb === 'function') {
  cb(null, file);
  called = true;
}
```

**Comment by odensc** [comment]
What's the line you're using for gulp.watch? And what gulp version are you using? Something might have changed.

**Comment by davej** [comment]
Gulp 3.9.0

``` javascript
gulp.task('watch', ['electron'], function() {
  gulp.watch(paths.sass, ['sass']);
  gulp.watch(paths.earlUI, ['earl-ui']);
  gulp.watch(paths.earlServices, ['earl-services']);
  gulp.watch([
    './services/js/*.js',
    './app/www/js/*.js',
    './app/www/**/*.html',
    './app/www/css/*.css'
  ], runElectron.rerun);
});
```

**Comment by odensc** [comment]
Something in `3.9.0` probably changed watch behavior, I'll commit & publish a fix ASAP.

**Comment by davej** [comment]
Thanks :+1:

**Comment by odensc** [comment]
Alright, I reproduced it with `gulp@3.9.0`. Fixed & published as [`1.2.1`](http://npm.im/gulp-run-electron). :+1:

**Comment by davej** [comment]
Yep, fix works!
